contrib/ansible/openshift/register.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+remote=$1
+group=$2
+default_tools_interval=$3
+ose_master_interval=$4
+ose_node_interval=$5
+tool_dir=/var/lib/pbench-agent/tools-default
+ansible_host=$6
+index=$7
+index=$(( index + 1 ))
+label=svt_"$group"_"$index"
+
+## check if a remote is already registered
+for tool in `/bin/ls $tool_dir`; do
+		remote_host=$(echo "$tool" | grep "^remote")
+		remote_hostname=`echo "$remote_host" | awk -F@ '{print $2}'`
+		if [ "$remote" == "$remote_hostname" ]; then
+                	truncate -s -1 $tool_dir/$tool
+                	printf _"$group"_"$index" >> $tool_dir/$tool
```

Comment:
Consider the case where etcd and master are same, in this case I want the label to be master_etcd, so I am using >> to append etcd label to master.
For each playbook run, stop tools, kill tools, clear tools commands are executed. This prevents this from not adding the same label a second time.
For the first run lets say it registers tools on master, the label will be svt_master, then it checks if any of the other nodes are same, if the nodes are same then it will just append the group like svt_master_etcd